pkg/util/orchestrator/pod.go

```diff
@@ -151,6 +151,7 @@ func extractPodMessage(p *v1.Pod) *model.Pod {
 	podModel.IP = p.Status.PodIP
 	podModel.RestartCount = 0
 	podModel.QOSClass = string(p.Status.QOSClass)
+	podModel.PriorityClass = string(p.Spec.PriorityClassName)
```

Comment:
seems that`PriorityClassName` https://github.com/kubernetes/api/blob/96087243fecba390391364dd42abe2f3ff21c1e3/core/v1/types.go#L3124 is already a string so the extra cast is not necessary

Reply:
Done